test_runner/regress/test_tenant_conf.py

```diff
@@ -18,7 +19,11 @@ def test_tenant_config(neon_env_builder: NeonEnvBuilder):
     neon_env_builder.pageserver_config_override = """
 page_cache_size=444;
 wait_lsn_timeout='111 s';
```

Comment:
```suggestion
page_cache_size=444
wait_lsn_timeout='111 s'
```

Are these semi-colons even part of toml? https://www.toml-lint.com/ does not like it.

But anyways, could align the `'` so the reader does not have to wonder if toml supports both. According to toml-lint.com that is fine. But with the semi-colons, I am wondering how did this ever work.

```suggestion
page_cache_size = 444
wait_lsn_timeout = "111 s"
```

Reply:
Also no semicolon (or 3b or 59) on https://github.com/toml-lang/toml/blob/1.0.0/toml.abnf or https://toml.io/en/v1.0.0.

Reply:
Not part of this PR, I don't like drive-by changes, they just clutter up the patch.

I'll fix it in another PR

Reply:
Oki, I still don't understand why this works but resolving.